### PR TITLE
Setup Hello Microverse!

### PR DESCRIPTION
**Pull request by ebidien**
### Setup Hello Microverse!
The following changes were made to the project in this feature branch:

- Add a `.gitignore` file including `node_modules` in it
- Set up a linter for HTML and CSS
- Add a descriptive `README.md` file using the **Microverse Students README Template**
- Add `index.html` file with the heading **"Hello Microverse!"**
- Add `styles.css` file with styles for the **`heading` class** in `index.html`
- Add `test.md` file and include in `.gitignore` 
- Edit the content of `README.md` template to suit the current project
- Install linters and fix linter errors
- Add omitted link to `styles.css` in `index.html`

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (Ibizugbe) [inline test.md]: It is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) in this project that the test.md file should be ignored by git😉.
testgitignore
Your test.md file is still on GitHub, which means it is still not being ignored. kindly fix this issue as it is a vital requirement to complete this milestone. 🤗🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @ebidien 👋, 

68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d


Good job so far!
There are still some issues that you still need to work on to go to the next project but you are almost there!😉

## Highlights
- [x] Linters have been correctly setup👍.
- [x] The repository looks professional👍.
- [x] The correct git-flow method has been used👍.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ebidien** [comment]
Hi @Ibizugbe, I have used `git rm --cached test.md` to remove the `test.md` file from the remote repository. Although it still exists in my local branch, it was ignored as expected in my last push to the remote branch. Thanks for clearly explaining the improvements needed. I hope this will be all.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,200 @@
-# hello-microverse
-First Microverse project to demonstrate effective use of linters, the GitHub flow, and professional documentation
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (rskinnerc) [inline README.md:5]: [OPTIONAL] Great job implementing the recommended template. Remember that you can always use your own logo or remove it altogether. 😋😋

File: README.md [hunk 3]
```diff
@@ -1,2 +1,200 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse! <a name="about-project"></a>
+
+> About Hello Microverse!
```
Comment (rskinnerc) [inline README.md]: [OPTIONAL] Those comments are there for your information and guidance on how o update the README file. For future projects, please consider removing them from the file. 🙂🙂

**Review by rskinnerc — APPROVED** [review]
Hi @ebidien 👋👋👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

## Highlights
- Linters correctly installed and running
- Good job using the recommended README.md file
- Great! You have fixed the test.md file as previously requested. Good idea to use the `git` command to remove the file from the cache
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (ebidien) [reply README.md]: @rskinnerc, thanks for the review! I will apply the optional recommendations in future projects. Thanks again.

**Pull request merged**